lib/stripe.rb

```diff
@@ -278,6 +281,11 @@ def self.authentication_error(error, resp, error_obj)
                             resp.headers)
   end
 
+  def self.rate_limit_error(error, resp, error_obj)
+    RateLimitError.new(error[:message], resp.code, resp.body, error_obj,
+                            resp.headers)
```

Comment:
nit: indentation. `resp.headers, should start at the same column as `error[:message]`